src/mantine-dates/src/components/Month/Month.test.tsx

```diff
@@ -36,8 +36,12 @@ describe('@mantine/core/Month', () => {
   });
 
   it('renders correct amount of days', () => {
-    const element = shallow(<Month month={new Date(2021, 1, 2)} />);
+    let element = shallow(<Month month={new Date(2021, 1, 2)} />);
```

Comment:
Do not reassign enzyme wrappers in tests, always create new variables with const:
```tsx
const monday = shallow(<Month month={new Date(2021, 1, 2)} />);
const sunday = shallow(<Month month={new Date(2021, 1, 2)} firstDayOfWeek="sunday" />);

// perform tests
```